cat-launcher/src-tauri/schemas/schema.sql

```diff
@@ -42,3 +42,12 @@ CREATE TABLE IF NOT EXISTS last_played_version (
     version TEXT NOT NULL,
     FOREIGN KEY (game_variant) REFERENCES variants (name)
 );
+
+-- This table stores the play time for each game variant and version.
+CREATE TABLE IF NOT EXISTS play_time (
+    id INTEGER PRIMARY KEY,
+    game_variant TEXT NOT NULL,
+    version TEXT NOT NULL,
+    duration_in_seconds INTEGER NOT NULL,
+    FOREIGN KEY (game_variant) REFERENCES variants (name) ON DELETE CASCADE
+);
```

Comment:
@google-labs-jules Look at the code to determine the access pattern on the paly time database. Use that to create an index that will improve performance.